src/core/payload.ts

```diff
@@ -113,6 +113,7 @@ async function* payloadToMultipartItr(
     const separator = enc.encode(`\r\n--${boundary}\r\n`);
     let first = true;
     for (const [key, value] of Object.entries(payload)) {
+        if (value == null) continue;
```

Comment:
Without trying this out… `parse_mode: undefined` shouldnt match `null`?

Maybe I overlooked something but is this working recursive into objects?

Reply:
> `parse_mode: undefined` shouldnt match `null`?

```
< undefined === null
> false
< undefined == null
> true
```

Welcome to JS.

> Maybe I overlooked something but is this working recursive into objects?

https://github.com/grammyjs/grammY/blob/446a02471594572ff253a43a01e174b4971a6d37/src/core/payload.ts#L117 calls https://github.com/grammyjs/grammY/blob/446a02471594572ff253a43a01e174b4971a6d37/src/core/payload.ts#L30 which already replaces `null` values by `undefined`, which will be removed during serialisation.

Reply:
> Welcome to JS.

Readable code is fun :D

Reply:
It's consistent with the rest of the ecosystem. It's also less verbose than `value === undefined || value === null` but I do see that point that it could be more readable to use the long version.